e2e-tests/tests/podsPage.spec.ts

```diff
@@ -0,0 +1,38 @@
+import { test } from '@playwright/test';
+import { HeadlampPage } from './headlampPage';
+import { podsPage } from './podsPage';
+
+test('multi tab create delete pod', async ({ browser, }) => {
+
+    // This test may be slow to create and delete a pod
+    test.setTimeout(60000);
+    const name = 'examplepodlol';
+
+    const instance1 = await browser.newContext();
+    const page1 = await instance1.newPage();
+    const window1 = new HeadlampPage(page1);
+    await window1.authenticate();
+
+    const page2 = await instance1.newPage();

```

Comment:
I think here it's supposed to be with a new instance? 
(So that there are two windows.)

Reply:
So newPage() makes a new tab, and that is good. 
(I was mistaken about what newPage does.)